k8s-path-cleanup.tf

```diff
@@ -76,28 +76,28 @@ resource "kubernetes_deployment" "beekeeper_path_cleanup" {
           }
 
           env {
-            name  = "AWS_REGION"
+            name  = local.aws_region_key
```

Comment:
why is this change required ?

Reply:
We put it to refactor the code a little. We saw that the rest were using locals and that one was the only one that didn't use it.

Reply:
ok.